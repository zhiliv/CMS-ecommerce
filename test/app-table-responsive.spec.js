import { shallowMount } from '@vue/test-utils'
import Row from '@/components/app/table-responsive/table-responsive.vue'

describe('app-table-responsive.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(Row, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        classes: 'simple',
        styles: 'background: white',
        id: 'indent',
        responsive: 'true'
      },
    })
  })
  it('Содержимое html для app-table-responsive ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
