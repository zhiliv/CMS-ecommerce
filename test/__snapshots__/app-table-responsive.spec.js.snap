// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-table-responsive.js Содержимое html для app-table-responsive  1`] = `
<div
  class="responsive simple"
  id="indent"
  style="background: white;"
/>
`;
